document/CoreFormat.mdk

```diff
@@ -678,26 +801,26 @@ A `result` object may contain a property named `toolFingerprint` whose value is
 that contributes to the process of uniquely identifying the result.
 [Annex A](#Annex-A) explains how a result management system can use this value.
 
-### `executionFlows` property { #result-executionFlows }
+### `codeFlows` property { #result-codeFlows }
 
-A `result` object may contain a property named `executionFlows` whose value is an array of arrays of
+A `result` object may contain a property named `codeFlows` whose value is an array of arrays of
 `annotatedCodeLocation` objects (&sect;[#annotatedCodeLocation]).
-For convenience, in this context we refer to an array of `annotatedCodeLocation` objects as an `executionFlow` object;
-hence we may say that the value of the `executionFlows` property is an array of `executionFlow` objects.
-The `executionFlows` property is intended for use by static analysis tools that perform flow analysis
+For convenience, in this context we refer to an array of `annotatedCodeLocation` objects as a `codeFlow` object;
+hence we may say that the value of the `codeFlows` property is an array of `codeFlow` objects.
+The `codeFlows` property is intended for use by static analysis tools that perform flow analysis
```

Comment:
it is better to focus on the reporting value rather than the analysis technique. 'flow analysis' means something specific to some audiences. but a tool based on abstract interpretation can make equal use of this mechanism. so, suggest reorienting the language and avoid asserting anything specific about the tool approach. 'is intended for use by static analysis tools that provide execution path details that illustrate a possible problem in code.' or some such.